README.md

```diff
@@ -110,6 +110,10 @@ const handleCheckbox = async (id, e) => {
 - We recommend [enabling console persistence](https://stackoverflow.com/questions/5327955/how-to-make-google-chrome-javascript-console-persistent) while developing replicache-enabled apps to make debugging easier.
 - Remember that data changes can happen "underneath" you and cause `subscribe()` to re-fire at any time. These changes can come from the server or from a different tab. If your UI is not reactive (driven solely by the data model) you need to take extra steps to ensure the UI is in sync with the data.
 
+## Versioning
```

Comment:
Is this really necessary? I feel like it's kind of expected that all js projects use semver.

Reply:
It's important that this document stay focused as it's the getting-started guide.

Reply:
Fair enough.

Reply:
Or sorry, stated differently: what was the motivation for this?

If we add this, can we maybe put it in tips, or at the bottom of the doc (so that the cute emoji progression is maintained), or else put it elsewhere (like HACKING.md, if it's meant more for the development team)?